CodeofConduct.md

```diff
@@ -0,0 +1,72 @@
+# 🌟 Code of Conduct
+
+## Our Pledge 🤝
+
+We as members, contributors, and leaders pledge to make participation in the ServiceNow Hacktoberfest community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Our Standards ⚖️
+
+Examples of behavior that contributes to a positive environment for our community include:
+
+- 🌼 Using welcoming and inclusive language
+- 🤝 Being respectful of differing viewpoints and experiences
+- 🎯 Gracefully accepting constructive criticism
+- 🌍 Focusing on what is best for the community
+- 💖 Showing empathy towards other community members
+- 🤗 Demonstrating empathy and kindness toward other people
+- 📝 Giving and gracefully accepting constructive feedback
+- 🙏 Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+
+Examples of unacceptable behavior include:
+
+- 🚫 The use of sexualized language or imagery, and unwelcome sexual attention or advances
+- 🗯️ Trolling, insulting, derogatory comments, and personal or political attacks
+- 🚷 Public or private harassment
+- 🔒 Publishing others' private information without their explicit permission
+- ⚠️ Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities 🛡️
+
+Project maintainers are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that is deemed inappropriate, threatening, offensive, or harmful.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct, and will communicate reasons for moderation decisions when appropriate.
+
+## Scope 🌐
+
+This Code of Conduct applies within all community spaces, and also applies when an individual is officially representing the ServiceNow Hacktoberfest community in public spaces. Examples of representing our community include using an official project email address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement 🚨
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at [earl.duque@servicenow.com](mailto:earl.duque@servicenow.com). All complaints will be reviewed and investigated promptly and fairly.
```

Comment:
Having Earl's email here is probably not a good or future-proof idea. Perhaps just provide the Slack channel, unless there's a better alternative?

Reply:
Lets use a reference to the `#hacktoberfest` channel, and if they are not comfortable with this public avenue, to reach out to a maintainer or lastly, to use Earl's email address until we can introduce a formal channel for that via a mailbox or outbound facing DL. @earlduque your thoughts?

Reply:
i don't mind having my email address because it's technically the most official thing we have now, i added the slack channel too though